spec/unit/transaction/report_spec.rb

```diff
@@ -392,15 +376,6 @@ def add_statuses(count, type = :file)
         @report.finalize_report
         expect(metric(:time, "foobar")).to eq(50)
       end
-
-      it "should have a total time" do
-        add_statuses(3, :file) do |status|
-          status.evaluation_time = 1.25
-        end
-        @report.add_times :config_retrieval, 0.5
-        @report.finalize_report
-        expect(metric(:time, "total")).to eq(4.25)
-      end
```

Comment:
Why did you remove these tests? It seems like we do still want to have a total time in the report, even though it's no longer being calculated in the same way. Maybe we just want to check that `total` is present in the report?

Reply:
I wasn't able to get this test to react to adding/removing the new total time entry here.

Reply:
Yeah, we probably don't have a total time in the report in these. This is fine